download_files.py

```diff
@@ -32,6 +32,17 @@ def _download_files(urls_and_outpaths: List[Tuple[str, str]], parent_dir: Path)
         if not outdir.exists():
             os.makedirs(outdir)
         subprocess.run(["wget", url, "-O", full_outpath])
+        # if downloading a zip file, unzip
+        if outpath.suffix == ".zip":
+            # unzip in parent directory
```

Comment:
so to clarify, the main benefit of doing this is it saves you (and whoever needs to update the images) a lot of time and manual effort since the alternative is to copy and paste each individual image path into `data_urls.csv`? Also i guess this keeps `data_urls.csv` cleaner.

Reply:
yep! maybe i'm dumb and there's a dropbox api that would allow me to easily get all the permalinks of files in a folder, but even then i think this is a bit easier to do